espree.js

```diff
@@ -785,7 +823,20 @@ function scanStringLiteral() {
             ch = source[index++];
             if (!ch || !syntax.isLineTerminator(ch.charCodeAt(0))) {
                 switch (ch) {
+                /*eslint-disable no-fallthrough */
                 case "u":
+                    // If string literal contains an ES6 extended unicode code point escape
+                    // handle it. Otherwise, fall through to scan the hex value.
+                    if (source[index] === "{") {
+                        if (extra.ecmaFeatures.unicodeCodePointEscapes) {
+                            ++index;
+                            str += scanUnicodeCodePointEscape();
+                        } else {
+                            throwError({}, Messages.UnexpectedToken, "ILLEGAL");
+                        }
+                        break;
+                    }
+                /*eslint-enable no-fallthrough */
```

Comment:
The formatted gotten borked in that comment, so I'm not entirely sure what you're asking. Any comment, line or block, with the words "falls through" are valid.